apps/coordinator/src/selectors/transaction.ts

```diff
@@ -138,36 +133,19 @@ export const extractSignaturesFromPSBT = (psbt: Psbt, inputs: Input[]) => {
 // SELECTORS
 // ====================
 
-/**
- * Extract input identifiers from PSBT (txid:index)
- */
 export const selectInputIdentifiersFromPSBT = createSelector(
   [(state: WalletState, psbt: Psbt) => psbt],
-  (psbt: Psbt) => {
-    return new Set(
-      psbt.txInputs.map((input) => {
-        return createInputIdentifier(
-          /*
-           * All input TXIDs are expected to be in **big-endian**
-           * format (human-readable format). Which we get from block explorers, wallets, APIs
-           * But PSBTs will need txid to be in little-endian format to ensure compatibility with Bitcoin's
-           * internal data structures and processing so here we convert the txid to little-endian format
-           */
-          convertTxidToLittleEndian(input.hash),
-          input.index,
-        );
-      }),
-    );
-  },
+  (psbt: Psbt) => getInputIdentifiersFromPsbt(psbt),
 );
 
 /**
  * Selector for "available inputs" from the unspent store
  * This is the first strategy - normal PSBT case where we get all the inputs we have in our wallet
  */
 export const selectAvailableInputsFromPSBT = createSelector(
-  [getSpendableSlices, selectInputIdentifiersFromPSBT],
-  (slices: any[], inputIdentifiers: Set<string>) => {
+  [getSpendableSlices, (state: WalletState, psbt: Psbt) => psbt],
```

Comment:
this selector needed to pass in the psbt to get the sequences as well. maybe there was another way to do this more idiomatically for reselect?